jenkins-jobs/jobs/EclipsePluginsJob.groovy

```diff
@@ -0,0 +1,95 @@
+setupReleaseBuild('release')
```

Comment:
minor quibble—`setup` here implies an imperative action, whereas its actually just a macro used for the declarative Jobs DSL